fleetspeak/src/server/components/proto/fleetspeak_components/config.proto

```diff
@@ -57,6 +57,11 @@ message Config {
   // with this server's notification port. Has no effect when
   // notification_listen_address is unset.
   string notification_public_address = 6;
+
+  // If set, a HTTP notifier implementation is used for sending notifications.
+  // Set this if running a pure admin server (without a notification listener)
+  // in a distributed setup.
+  bool notification_use_http_notifier = 10;
```

Comment:
Is there any practical use-case for setting `notification_use_http_notifier` to false? Can we simplify the config by always defaulting to http notifier if `https_config` is not set?

Reply:
I'd thought about it, but I'd prefer to make this configuration explicit. Because:

Setting this to `true` constitutes a rather specialized use case (the common cases being non-distributed setup, distributed setup where we run both http and admin server).

I also get confused, if activating a certain combination of components causes implicit activation of others.

I think in this specialized use case it makes sense to explicitly activate usage of either of the http notification server-side and client-side, also to make clear what is actually going on (what is sending notifications to what).

Reply:
Got it, makes sense. One more question. Would it make sense to put this parameter into `AdminConfig`? From the code it seems that it's only used in the admin server.

Reply:
My thought was that it should be grouped with the other `notification_` config options, since it configures the notifications "component". (Ideally, all 3 `notification_` options should be in their own "section").

Reply:
Agreed. Let's keep them here and refactor into a separate group later.